app/src/sharedTest/java/org/oppia/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -156,7 +161,10 @@ class TopicLessonsFragmentTest {
 
   @Test
   fun testLessonsPlayFragment_loadRatiosTopic_completeStoryProgress_isDisplayed() {
-    storyProgressTestHelper.markFullStoryPartialTopicProgressForRatios(profileId, /* timestampOlderThanAWeek= */ false)
+    storyProgressTestHelper.markFullStoryPartialTopicProgressForRatios(
+      profileId, /* timestampOlderThanAWeek= */
+      false
```

Comment:
Nit change here and everywhere else in this file:
```
      profileId,
      /* timestampOlderThanAWeek= */ false
```

Reply:
Done